fastly/block_fastly_service_v1_logging_kafka.go

```diff
@@ -93,7 +93,38 @@ func (h *KafkaServiceAttributeHandler) Register(s *schema.Resource) error {
 		"tls_hostname": {
 			Type:        schema.TypeString,
 			Optional:    true,
-			Description: "The hostname used to verify the server's certificate. It can either be the Common Name or blockAttributes Subject Alternative Name (SAN).",
+			Description: "The hostname used to verify the server's certificate. It can either be the Common Name or a Subject Alternative Name (SAN).",
+		},
+
+		"parse_log_keyvals": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: "Enables parsing of key=value tuples from the beginning of a logline, turning them into record headers.",
+		},
+
+		"request_max_bytes": {
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Description: "Maximum size of log batch, if non-zero. Defaults to 0 for unbounded.",
+		},
+
+		"auth_method": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "SASL authentication method. One of: plain, scram-sha-256, scram-sha-512.",
+		},
+
+		"username": {
```

Comment:
This doesn't align with either the API documentation nor the Go-Fastly response struct tag, which has it defined as `user`.

Reply:
:+1: Done